macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -138,13 +137,19 @@
                 bucket_start,
                 bucket_end,
                 metric_value,
-                case 
+                case
                     when training_stddev is null then null
+                    {% if anomaly_direction is not none %}
```

Comment:
I think it's better to avoid these conditions in the query itself, as it is already hard to debug and understand it. 
Maybe set a default value of 'Both' to anomaly direction? 
It's also more explicit, is this is in fact the default behavior